miner/bid_simulator.go

```diff
@@ -34,6 +36,8 @@ const (
 	leftOverTimeRate = 11
 	// leftOverTimeScale is the scale of left over time to simulate a bid
 	leftOverTimeScale = 10
+
+	PayBidTxGas uint64 = 21000 // The gas reserved for pay bid tx
```

Comment:
can you check the code of builder, I think it is 25000.

Reply:
fixed